### DISCUSSION TITLE
Advanced Delta Tuning?

### DISCUSSION THREAD

**Discussion by wagzledz** [phnxdrgn536/FLSun-v400#32] [Q&A]
So first off, thanks phnxdragon for and outstanding GitHub collection of info.   kudos sir.  

Secondly, im a n00b to the delta life.   the v400 looks and acts promising.  using basic Cura profile it came with (slightly tweaked) and im getting rather accurate prints.     my overall object size is off by about -.10 on stuff. cant complain considering its screaming along with 400mms in the main speed setting. 

Im here mainly because the z hop function doesn't seem to do anything in Cura?   I too get nozzle drag when running fast etc.  I haven't ripped any prints off the bed yet.. but im sure it'll happen lol 

my 2 questions are, does doing the advanced tuning help solve my slight dimensional accuracy issue?     And does it by chance also help stop the nozzle drag and allow the z hop to actually work? 

ive gone as far as making my z-hop ungodly high.. (5 mm..) to see if it'll not rub... with no such luck.  so I guess that kinda tells me that z hop isn't in fact working like normal?  (although these aren't normal printers either..). 

Fill me in with words of wisdom and hope.  lol   what am I missing?  Thanks folks

-WagZ

**Comment by phnxdrgn536** [comment]
Ok so Z-hop does not occur during normal printing only during certain
travel moves. Three things to eliminate dragging are the enhanced delta
calibration, e-step calibration, and adjust your infill overlap. The last
one is a big thing… the infill overlap is essentially the amount that the
infill overlaps into the walls. I believe the default is 10% I lowered mine
to about 8%… if you look during printing there are small “bubbles” of
material on the walls where the infill joins and this is the main cause of
dragging… reducing the infill overlap reduces these blobs and lowers the
dragging issue.

**Reply by wagzledz** [reply]
Thanks boss.   ill give it a shot.  My rotation distance is already adjusted properly.  I also got flow turned down to 95 and infill at 10% so ill see what dropping it to 8% does.   

Also  just downloaded your profiles too so ill give them a shot also.    im still using FLSUN firmware on the klipper pad.   I dont wanna change it just yet incase something catastrophic goes wrong. lol   im pretty new to klipper but im learning.
